src/components/Notification/NotifDrawer/NotifDrawerTitle.js

```diff
@@ -0,0 +1,49 @@
+import ClassNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const NotifDrawerTitle = ({
+  onCloseClick,
+  onExpandClick,
+  title,
+  children,
+  className,
+  ...props
+}) => {
+  const classes = ClassNames('drawer-pf-title', className);
+
+  return (
+    <div className={classes} {...props}>
+      <a
```

Comment:
maybe use icon component? also is it possible to have the expansion optional ? (via a prop?)

Reply:
The expand toggle already has an icon in the `<a>` class, i added an icon for the close button.
as for the expansion, there is a prop for it in the `NotifDrawer`, I'll tag you.